lib/sourcemapper.js

```diff
@@ -56,25 +56,31 @@ function processSourcemap(infoMap, mapPath, callback) {
   // this handles the case when the path is undefined, null, or
   // the empty string
   if (!mapPath || !_.endsWith(mapPath, MAP_EXT)){
-    return callback(new Error('The path ' + mapPath +
-                    ' does not specify a sourcemap file'));
+    return setImmediate(function() {
+      callback(new Error('The path ' + mapPath +
+        ' does not specify a sourcemap file'));
+    });
   }
   mapPath = path.normalize(mapPath);
 
   fs.readFile(mapPath, 'utf8',
     function(err, data) {
       if (err){
-        return callback(new Error('Could not read sourcemap file ' + mapPath +
-                        ': ' + err));
+        return setImmediate(function() {
```

Comment:
It is not necessary to use `setImmediate` here as this is already async (called after `readfile`).